templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver {{ driver.username }}</h1>
+  <p>Are you sure you want to delete {{ driver.first_name }} {{ driver.last_name }} from database?</p>
+  <form action="" method="post">
```

Comment:
The form element is missing an 'action' attribute. You should specify the URL for the form submission, such as `{% url 'taxi:driver-delete' pk=driver.id %}`, to ensure the form data is sent to the correct view.